atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableMetadata.java

```diff
@@ -83,6 +83,15 @@ public LogSafety getNameLogSafety() {
         return LogSafety.UNSAFE;
     }
 
+    /**
+     * @return whether the table has densely accessed wide rows. This helps identify tables which could benefit from
+     * setting database tuning parameters to values more in line with such workflows, like the _transactions2 table.
+     */
+    @Value.Default
+    public boolean hasDenselyAccessedWideRows() {
```

Comment:
We should either not persist this when default, or make sure to change the CassandraKVSImpl table creation code which currently updates table metadata for a table if the entry persisted in the _metadata table differs from the new serialized table metadata. As is, this would make us rewrite table metadata for all tables in our schemas